cluster/test/commands_on_keys_test.rb

```diff
@@ -47,7 +47,6 @@ def test_migrate
   def test_object
     redis.lpush('mylist', 'Hello World')
     assert_equal 1, redis.object('refcount', 'mylist')
-    assert_equal 'quicklist', redis.object('encoding', 'mylist')
```

Comment:
It was changed to 'listpack' in Redis 7.2. We have no advantages for testing redis minor behavior, so just removed.